gen/tollvm.cpp

```diff
@@ -274,6 +282,18 @@ bool needsCOMDAT() {
   return global.params.targetTriple->isOSBinFormatCOFF();
 }
 
+bool supportsCOMDAT() {
+  const auto &triple = *global.params.targetTriple;
+  return !(triple.isOSBinFormatMachO() ||
+#if LDC_LLVM_VER >= 500
+           triple.isOSBinFormatWasm()
+#else
+           triple.getArch() == llvm::Triple::wasm32 ||
+           triple.getArch() == llvm::Triple::wasm64
+#endif
+  );
```

Comment:
```suggestion
  return !(triple.isOSBinFormatMachO() || triple.isOSBinFormatWasm());
```

Even if `isOSBinFormatWasm` was added in llvm 5, ldc only supports >=llvm-15 so `LDC_LLVM_VER` will always be `>= 1500`

Reply:
This was picked from older code, before it got removed. I'll remove this, since llvm-5 is no longer supported by LDC.